proposals/devcontainer-distribution.md

```diff
@@ -0,0 +1,71 @@
+# Community creation of templates and features and their distribution.
+
+This proposal is created to arrive at a schema where different communities and organizations can create features and devcontainer templates and make them discoverable to users.
+
+The proposal centers on the following ideas:
+
+- Discoverability for users both on the web (github.com) and on any tools that implement the [Dev container specification](../docs/specs/devcontainer-reference.md).
+- Users can validate the origin of the template or feature that they want to use.
+- Users can pin a particular version of a feature to allow consistent repeatable environments.
+
+# Proposal
+
+## Github Topics
+
+This can be achieved with the use of [GitHub Topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics), specifically the [devcontainers](https://github.com/topics/devcontainers) topic. Additionally the repositories can be marked with **devcontainers-templates** and/or **devcontainers-features** to mark specifically the contents of the repository.
```

Comment:
This starts by saying "This can be achieved..." - what is the "This" we're trying to achieve? And do we want to more concretely guide users to `devcontainers-templates` and `-features`, so there's a clear path to discovery (rather than multiple options), and users who want others in the community to easily find their repos can do so clearly? 

Perhaps something like the following would work:

```suggestion
A set of discoverable dev container-related repos can be achieved with the use of [GitHub Topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics). You may explore repos that GitHub has automatically considered part of the [devcontainers](https://github.com/topics/devcontainers) topic. If you'd like your template or feature repo to be easily discoverable to the community, you may add the **devcontainers-templates** and/or **devcontainers-features** topic. You may read more on how to do this in the [topics guide](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics).
```